docs/design/scheduler-simulate-design.md

```diff
@@ -0,0 +1,32 @@
+# Volcano scheduler simulate
+## background
+* Consider such situation: users changed the parameter of nodeorder plugin and need to know the effect to the production enviroment. For example, after change the mostrequested.weight, if the average wait time of big task is shorter than before, etc.
```

Comment:
I think the simulation scheduling scenario is more suitable for node simulation. (e.g., we don’t have GPU and NPU nodes, if there is some bugs in GPU or NPU scheduling, we can use simulation scheduling to debug. We do meet this scenario in our production). And to test the performance of the scheduler in large-scale clusters. If it is just for feature validation, there is no much difference with starting a kind cluster.

Reply:
we can add more node features of gpu in nodes.csv like gpu_allocatable, etc